internal/db/profile_selector_scan_test.go

```diff
@@ -47,6 +47,20 @@ func TestScan(t *testing.T) {
 				Comment:  "comment1",
 			},
 		},
+		{
+			name:  "Valid input with commas in the selector",
+			input: []byte(fmt.Sprintf("(%s,%s,repository,\"repository.properties['github/primary_language'] in ['TypeScript', 'Go']\",\"comment1\")", selectorId, profileId)),
```

Comment:
Do you want to use ` for this string to avoid needing to escape single and double quotes?

Reply:
will do!

Reply:
done!